cloud/docker-image/pom.xml

```diff
@@ -254,17 +254,89 @@
+                            <include>io/aklivity/zilla/engine/**</include>
+                            <include>io/aklivity/zilla/binding-*/**</include>
+                            <include>io/aklivity/zilla/catalog-*/**</include>
+                            <include>io/aklivity/zilla/exporter-*/**</include>
+                            <include>io/aklivity/zilla/guard-*/**</include>
+                            <include>io/aklivity/zilla/metrics-*/**</include>
+                            <include>io/aklivity/zilla/validator-*/**</include>
+                            <include>io/aklivity/zilla/vault-*/**</include>
+                            <include>io/aklivity/zilla/command/**</include>
+                            <include>io/aklivity/zilla/command-*/**</include>
+                            <include>io/aklivity/zilla/manager/**</include>
+                            <include>io/aklivity/zilla/incubator/**</include>
+                            <include>org/agrona/**</include>
+                            <include>org/apache/apache/**</include>
+                            <include>org/apache/avro/**</include>
+                            <include>org/apache/maven/maven/**</include>
+                            <include>org/apache/maven/maven-parent/**</include>
+                            <include>jakarta/json/**</include>
+                            <include>jakarta/inject/**</include>
+                            <include>org/leadpony/justify/**</include>
+                            <include>org/leadpony/joy/**</include>
+                            <include>org/snakeyaml/snakeyaml-engine/**</include>
+                            <include>org/eclipse/yasson/**</include>
+                            <include>org/glassfish/**</include>
+                            <include>com/github/rvesse/**</include>
+                            <include>com/guicedee/**</include>
+                            <include>org/eclipse/ee4j/**</include>
+                            <include>com/ibm/icu/**</include>
+                            <include>org/bitbucket/b_c/jose4j/**</include>
+                            <include>org/slf4j/**</include>
+                            <include>org/antlr/**</include>
+                            <include>org/sonatype/oss/**</include>
+                            <include>com/fasterxml/oss-parent/**</include>
+                            <include>com/fasterxml/jackson/**</include>
+                            <include>org/yaml/snakeyaml/**</include>
+                            <include>org/junit/**</include>
+                          </includes>
+                        </fileSet>
+                      </fileSets>
+                    </inline>
+                  </assembly>
+                </assemblies>
+              </build>
+            </image>
             <image>
               <alias>zilla</alias>
-              <name>ghcr.io/aklivity/zilla:%v</name>
+              <name>ghcr.io/aklivity/zilla</name>
```

Comment:
We can restore the `%v` now given the feedback for the top level pom proposed changes I think.